db/schema.rb

```diff
@@ -41,52 +41,52 @@
   add_index "builds", ["project_id"], name: "index_builds_on_project_id", using: :btree
 
   create_table "commands", force: :cascade do |t|
-    t.text     "command",    limit: 10485760
+    t.text     "command",    limit: 16777215
```

Comment:
this is weird, the limit is set to `10.megabytes` which is 10485760 but that probably assuming non-unicode encoding anyway